roles/sosreport/defaults/main.yml

```diff
@@ -2,36 +2,10 @@
 sosreport_unarchive: false
 sosreport_tmpdir: /tmp/sosreport
 sosreport_cmd: sosreport
-sosreport_required_packages:
-  - sosreport
+sosreport_required_packages: ""
```

Comment:
not sure if we need this if this is present in /vars

Reply:
probably not

Reply:
cf. molecule/delegated/tests/sosreport.py